DeepSpeech.ipynb

```diff
@@ -1105,8 +1306,123 @@
    },
    "outputs": [],
    "source": [
-    "train()"
+    "# Test network\n",
+    "test_decodings, test_labels = forward(session, ted_lium.test)\n",
+    "_, test_wer = print_wer_report(session, \"Test\", test_decodings, test_labels)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Logging Hyper Parameters and Results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, as training and test are done, we persist the results alongside with the involved hyper parameters for further reporting."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "with open('%s/%s' % (log_dir, 'hyper.json'), 'w') as dump_file:\n",
```

Comment:
Already had a similar doubt about this. However: The only way I see is to have all the parameters already defined within a dictionary. But this would hinder us from defining them in a distributed way across the document.